core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -540,8 +540,8 @@ object KafkaConfig {
   val RackDoc = "Rack of the broker. This will be used in rack aware replication assignment for fault tolerance. Examples: `RACK1`, `us-east-1d`"
   /** ********* Log Configuration ***********/
   val NumPartitionsDoc = "The default number of log partitions per topic"
-  val LogDirDoc = "The directory in which the log data is kept (supplemental for " + LogDirsProp + " property)"
-  val LogDirsDoc = "The directories in which the log data is kept. If not set, the value in " + LogDirProp + " is used"
+  val LogDirDoc = "The directory in which the log data is kept (supplemental for " + LogDirsProp + " property). Default directory (" + Defaults.LogDir + ") is not safe for production deployment. Use valid persistent directory."
+  val LogDirsDoc = "The directories in which the log data is kept. If not set, the value in " + LogDirProp + " is used. Default directory (" + Defaults.LogDir + ") is not safe for production deployment. Use valid persistent directory."
```

Comment:
Is there a default? I thought the issue was the server.properties file.

Reply:
We have default for "log.dir". There is no default for "log.dirs".  If "log.dirs" is not set, we will use "log.dir"

Reply:
You're right. It's even worse than I thought. It would be good to remove this unsafe default.

Reply:
maybe we can merge this PR.  I will raise a minor KIP (if required) to remove the default .